packages/flutter/lib/src/gestures/arena.dart

```diff
@@ -275,3 +275,15 @@ class GestureArenaManager {
     return true;
   }
 }
+
+/// An arena for use with pointer gestures (e.g., scroll wheel events).
+///
+/// Pointer gestures are immediate, so are always resolved at sweep time. This
+/// means that [hold] and [release] are no-ops.
+class PointerGestureArenaManager extends GestureArenaManager {
```

Comment:
Would it make more sense to make this an independent class, unrelated to gesture arenas? It seems like we don't really need much of the arena machinery.

Reply:
Done.

You suggested just using a `List`, but since there needs to be a map of lists, a way for clients to register for callbacks if they win, and resolution step with cleanup, I ended up making a simplified arena class structure for these events. Since that structure is the same at a high level, it seemed like it made sense to share terminology. If you think that's confusing I can adjust.